modules/performanceplatform/manifests/checks/backups.pp

```diff
@@ -23,4 +25,37 @@
       handlers => ['default'],
       require  => File[$freshness_script],
     }
+
+    $graphite_fqdn = regsubst($fqdn, '\.', '_', 'G')
+    $graphite_disk = regsubst(regsubst($disk, '^/', ''), '/', '-', 'G')
+
+    performanceplatform::checks::graphite { "check_low_disk_space_${graphite_fqdn}_${graphite_disk}":
```

Comment:
Can we update performanceplatform::checks::disk and pass through parameters for critical and warning rather than duplicating it here? They can default to what they are currently.